LibSource/StereoOscillator.h

```diff
@@ -0,0 +1,64 @@
+#ifndef __QUADRATURE_OSCILLATOR_H__
```

Comment:
I think I'd rather see a ComplexOscillator

Reply:
I can rename it and I've considered that name. However that would inevitably cause confusion due to common misnaming of Buchla Complex Waveform generator as "complex oscillator". OTOH, it already is the correct name for some oscillators i.e. http://www.doepfer.de/A1104.htm

So just confirm if I should rename it.

Reply:
Yeah I meant renaming StereoOscillator => ComplexOscillator
and maybe changing that header guard...

Reply:
Guard should be changed for sure. I've ended up using that name because we can have a stereo oscillator that is not outputting signal in quadrature (to be visualized in 2D). Stereo oscillator osc/template just deals with 2 channels of audio, so I've decided that a more generic name is more suitable.

But ok, I can rename it and add a comment to explain that there's no Buchla in that class ;-)